src/home_robot/home_robot/control/goto_controller.py

```diff
@@ -26,11 +55,10 @@ class GotoVelocityController:
 
     def __init__(
         self,
-        cfg_name: Optional[str] = None,
+        cfg: Optional["DictConfig"] = None,
```

Comment:
why?

Reply:
Habitat-lab uses hydra for configuration.
Passing in cfg_name uses the home_robot configuration system, which currently is a minimal placeholder I created.
Changing the input to a DictConfig will allow for config propagation from Habitat-Lab